### DISCUSSION TITLE
v0.4.0

### DISCUSSION THREAD

**Discussion by reupen** [reupen/svg-services#137] [Announcements]
# Changes

- [resvg](https://github.com/RazrFalcon/resvg) was updated from version 0.36.0 to 0.43.0, including improvements to SVG support.
- The component is now compiled with Visual Studio 2022 17.11 and Rust 1.80.1.

<hr /><em>This discussion was created from the release <a href='https://github.com/reupen/svg-services/releases/tag/v0.4.0'>v0.4.0</a>.</em>

**Comment by marc2k3** [comment]
I don't know if you'll care or not but this latest version requires at least Windows 10 whereas previous versions did not.

rust 1.76.0 and later dropped for support for older versions of windows and you end up with this which means the DLL fails to load...

![image](https://github.com/user-attachments/assets/a0e16686-15b4-4818-b773-9e7b4127066a)

I worked around this by using 1.75.0

https://github.com/jscript-panel/resvg/commit/dd610bb85e7cb93c4787bf0be9f10f85b1ac3cf1

resvg itself only requires 1.67.1 for now.

If you stick to requiring10, that's fine by me. Just thought I'd mention it as I had a report about it for one of my components.

**Reply by reupen** [reply]
Hmm, I hadn't realised, thanks.

From https://blog.rust-lang.org/2024/02/26/Windows-7.html and https://blog.rust-lang.org/2024/05/02/Rust-1.78.0.html#compatibility-notes it sounds like they ended up making the change in 1.78... (And there is a way to target Windows 7 still, but you have to use a nightly version of Rust or build it yourself, or something....)

Anyway, I don't think I declare compatibility with any specific versions of Windows anywhere for this component 😄 I'll just add a note to the latest release notes mentioning that Windows 10 is now required.

**Reply by marc2k3** [reply]
Ah thanks for the correct info, I must have been looking at a non authoritative source about versions. 😬
